server/opts.go

```diff
@@ -316,6 +317,49 @@ type WebsocketOpts struct {
 	HandshakeTimeout time.Duration
 }
 
+// MQTTOpts are options for MQTT
+type MQTTOpts struct {
+	// The server will accept MQTT client connections on this hostname/IP.
+	Host string
+	// The server will accept MQTT client connections on this port.
+	Port int
+
+	// If no user name is provided when a client connects, will default to the
+	// matching user from the global list of users in `Options.Users`.
+	NoAuthUser string
+
+	// Authentication section. If anything is configured in this section,
+	// it will override the authorization configuration of regular clients.
+	Username string
+	Password string
+	Token    string
+
+	// Timeout for the authentication process.
+	AuthTimeout float64
```

Comment:
This b/c of mqtt protocol? vs `time.Duration`.

Reply:
No, simply to be consistent with all the auth timeout that we have in all other configs where we have auth block.